spec/github_helper_spec.rb

```diff
@@ -138,4 +138,119 @@ def mock_comment(body: '<!-- REUSE_ID: test-id --> Test', user_id: 1234)
       instance_double('Comment', id: 1234, body: body, user: instance_double('User', id: user_id))
     end
   end
+
+  describe 'get_milestone' do
+    let(:test_repo) { 'repo-test/project-test' }
+    let(:release_name) { '10.0' }
+    let(:client) do
+      instance_double(
+        Octokit::Client,
+        list_milestones: []
+      )
+    end
+
+    before do
+      allow(described_class).to receive(:client).and_return(client)
+    end
+
+    it 'receives the correct repository' do
+      expect(client).to receive(:list_milestones).with(test_repo)
+      described_class.get_milestone(test_repo, release_name)
+    end
+
+    it 'returns nil when no milestone exists' do
+      expect(described_class.get_milestone(test_repo, release_name)).to be_nil
+    end
+
+    it 'returns milestone from repo' do
+      allow(client).to receive(:list_milestones).and_return([{ title: 'release/10.0' }])
+      expect(described_class.get_milestone(test_repo, release_name)).to be_nil
+    end
+  end
+
+  describe 'create_milestone' do
+    let(:test_repo) { 'repo-test/project-test' }
+    let(:test_milestone_number) { '10.0' }
+    let(:test_milestone_duedate) { '2022-10-22T23:39:01Z' }
+    let(:client) do
+      instance_double(
+        Octokit::Client,
+        create_milestone: nil
+      )
+    end
+
+    before do
+      allow(described_class).to receive(:client).and_return(client)
+    end
+
+    it 'has the correct dates to code freeze without submission' do
+      comment = 'Code freeze: October 22, 2022 App Store submission: November 15, 2022 Release: October 25, 2022'
```

Comment:
Fun coincidence that I've changed that copy very recently during https://github.com/wordpress-mobile/release-toolkit/pull/419, which now makes this text contain newlines. You might want to update your fork's `trunk` branch with our latest `trunk` of the main repo, and merge `trunk` back into your PR's branch, to get the change and ensure that the new test you introduce will take the latest actual copy (with newlines) into consideration 🙂  (otherwise that test will pass in your fork… but fail as soon as we merge your PR in our own `trunk` branch)

Reply:
Yeah, I notice that when I was opening the new PR with the tests. 
I hope they indeed fail, so I will confirm that I write them right! 😁